packages/wonder-blocks-popover/src/components/__tests__/popover.test.tsx

```diff
@@ -602,19 +606,59 @@ describe("Popover", () => {
             );
 
             // Act
-            // Open the popover
-            const openButton = await screen.findByRole("button", {
-                name: "Open default popover",
-            });
-
-            await userEvent.click(openButton);
-            const popover = await screen.findByRole("dialog");
+            await userEvent.click(
+                await screen.findByRole("button", {
+                    name: "Open default popover",
+                }),
+            );
 
             // Assert
+            expect(
+                screen.getByLabelText("Popover Aria Label"),
+            ).toBeInTheDocument();
+
+            expect(
+                await screen.findByRole("dialog", {
+                    description: "This is a popover description",
```

Comment:
Shouldn't we be checking the title of the dialog is "This is a popover title"?

Reply:
I updated this test to be more consistent with the other tests, hopefully the intent is more clear now; since I'm using an aria label in this scenario, I would expect the dialog title to be `Popover Aria Label`